### PR TITLE
changes cluster_name to cluster_id in the request path

### LINKED ISSUE #1791: change `cluster_name` to `cluster_id` in API path
TBD

### PR DESCRIPTION
**Pull request by p0lyn0mial**
**What this PR does / why we need it**: simply changes cluster_name to cluster_id in the request path

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #1791 

**Documentation**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: api/pkg/handler/cluster.go [hunk 1]
```diff
@@ -719,10 +719,10 @@ func newDecodeGetClusterReq(c context.Context, r *http.Request) (interface{}, er
 	return req, nil
 }
 
-func decodeClusterName(c context.Context, r *http.Request) (string, error) {
-	clusterName := mux.Vars(r)["cluster_name"]
+func decodeClusterID(c context.Context, r *http.Request) (string, error) {
+	clusterName := mux.Vars(r)["cluster_id"]
```
Comment (glower) [inline api/pkg/handler/cluster.go]: please change this to clusterID as well

File: api/pkg/handler/request.go [hunk 2]
```diff
@@ -80,7 +80,12 @@ func decodeLegacyClusterReq(c context.Context, r *http.Request) (interface{}, er
 
 func decodeClusterReq(c context.Context, r *http.Request) (interface{}, error) {
 	var req GetClusterReq
-	req.ClusterName = mux.Vars(r)["cluster"]
```
Comment (p0lyn0mial) [inline api/pkg/handler/request.go:83]: I think that `metrics` and `clusters` ep didn't work.

**Review by glower — DISMISSED** [review]
one small change
LGTM

**Comment by p0lyn0mial** [comment]
@glower comment addressed, PTAL.

**Review by glower — APPROVED** [review]

**Pull request merged**